fe/src/composables/useUtils.ts

```diff
@@ -20,5 +20,13 @@ export const useUtils = () => {
     return start + "..." + end;
   };
 
-  return { truncate, middleTruncate };
+  const getDaysLeft = (dateInt: BigInt): number => {
+    const dateNow = new Date();
+    const dateNew = new Date(Number(dateInt) * 1000);
+    const timeDiff = dateNew.getTime() - dateNow.getTime();
```

Comment:
put type on every variables
```suggestion
    const dateNow: Date = new Date();
    const dateNew: Date = new Date(Number(dateInt) * 1000);
    const timeDiff : Date = dateNew.getTime() - dateNow.getTime();
```